### DISCUSSION TITLE
Schwierigkeiten mit Autodrive einen fertigen Kurs einzubinden!?!?! Sorry ich bin 68 Jahre alt und mein Englisch ist wirklich nicht gut...!

### DISCUSSION THREAD

**Discussion by Toaogurch** [community/community#86740] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hallo zusammen,
ich möchte bei der NF Marsch einen erstellten Kurs bei Autodrive einbinden. Ich spiele auf einem iMac. 
Kann mir einer sagen wie ich das hin bekomme...?!?!?!

**Comment by queenofcorgis** [comment]
Hi @Toaogurch , thanks for participating!
We take our [Code of Conduct](https://github.com/community/community/blob/0ca37cf594e438fd4aec3c7f3736b95e6b0b3567/CODE_OF_CONDUCT.md) very seriously and want to help ensure that everyone has a good experience free of antagonism and harassment. Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be locking any posts in languages other than English for now, including this one.
